ai-machine-learning/prometheus-timeseries-forecasting/02-accessing-the-web-console.md

```diff
@@ -0,0 +1,16 @@
+To access the OpenShift console:
```

Comment:
The users don't really need to access the Openshift console, can you change this to step 3 an "extra" step to troubleshoot if the backend isn't working